core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -1276,6 +1277,32 @@ public boolean isNeededDependenciesCompatibleWithInstalledVersion(PluginManager.
             return getDependenciesIncompatibleWithInstalledVersion(cache).isEmpty();
         }
 
+        /**
+         * Returns true if and only if this update addressed a currently active security vulnerability.
+         *
+         * @return true if and only if this update addressed a currently active security vulnerability.
+         */
+        @Restricted(NoExternalUse.class) // Jelly
+        public boolean fixesSecurityVulnerabilities() {
```

Comment:
NIT: Not used to such naming convention. I generally expect method starting with an action verb to do something. In this case if I call this method I expect it to fix the vulnerabilities.

💡 containsSecurityVulnerabilitiesFixes ?

Reply:
Not a problem while restricted